azure/datalake/store/enums.py

```diff
@@ -12,3 +12,9 @@ class ExpiryOptionType(Enum):
     relative_to_now = "RelativeToNow"
     relative_to_creation_date = "RelativeToCreationDate"
     absolute = "Absolute"
+
+
+class SyncFlac(Enum):
```

Comment:
Are you using this anywhere? Your actual usage above seems to use the strings directly.